setup.py

```diff
@@ -126,7 +126,7 @@ def run(self):
 
         "psutil == 2.1.2",
         "netifaces >= 0.8",
-        "ipaddr == 2.1.10",
+        "ipaddr == 2.1.11",
```

Comment:
The JSON release url metadata is not available for this release https://pypi.python.org/pypi/ipaddr/2.1.10/json - see "2.1.10" and "2.1.11" under "releases".